src/components/ResourceList/components/FilterControl/components/DateSelector/DateSelector.tsx

```diff
@@ -408,8 +411,12 @@ function getDateFilterOption(
   return filterValue;
 }
 
-function formatDateValue(date: Date) {
-  return date.toISOString().slice(0, 10);
+function formatDateForTimezone(date: Date) {
+  const tzOffsetTime = new Date(
+    date.getTime() - date.getTimezoneOffset() * 60000,
```

Comment:
Would also be nice to move `60000` into a constant to lend some clarity as to what exactly it is.